monai/data/image_reader.py

```diff
@@ -788,30 +793,66 @@ def get_data(
 
         return patches, metadata
 
+    def _evaluate_level(self, img, level):
```

Comment:
I feel the function name is a little bit confusing, maybe just call it `_check_level`?

Thanks.